xwiki-platform-core/xwiki-platform-security/xwiki-platform-security-authorization/xwiki-platform-security-authorization-api/src/main/java/org/xwiki/security/authorization/AuthorizationManager.java

```diff
@@ -48,6 +49,21 @@ public interface AuthorizationManager
      */
     String SUPERADMIN_USER = "superadmin";
 
+    /**
+     * Check whether the current user is {@code superadmin}.
+     *
+     * @param user a user reference
+     * @return {@code true} if the current user is {@code superadmin}, {@code false} otherwise
+     * @since 17.2.0RC1
+     * @since 16.10.5
+     * @since 16.4.7
+     */
+    @Unstable
+    default boolean isSuperAdmin(DocumentReference user)
+    {
+        return false;
```

Comment:
I'm a bit rusty on the topic of interfaces retro-compatibility, checking if it's best to:
1. stay like this
2. throw an exception
3. acknowledge the breakage in revapi configs

Reply:
Actually the implementation from `DefaultAuthorizationManager` can easily be moved as the default implem here.
Also, I've quickly listed all the places I could find that are using duplicated (and inconsistent) code to check for super admin. It's probably better to fix all of those at the same time, and therefore make sure that the single source of truth is on a module that can be used by all of the listed modules.